releasenotes/notes/fix-callback-multi-circuits-f8444aef40c7bd89.yaml

```
@@ -0,0 +1,8 @@
+---
+fixes:
+  - |
+    Fixed a bug in ``PassManager.run()`` where the ``callback`` function was not
```

```suggestion
    Fixed a bug in :meth:`.PassManager.run` where the ``callback`` function was not
```

This will create a cross-reference link to the method.